docs/content/mongocxx-v3/working-with-bson.md

```diff
@@ -39,8 +39,8 @@ list builder:
 
 ```c++
 // { "hello": "world" }
-bsoncxx::builder::list list_builder = {"hello", "world"};
-bsoncxx::document::view document = list_builder.view().get_document();
+bsoncxx::builder::list::document list_builder = {"hello", "world"};
```

Comment:
I suggest showing multiple fields, or an embedded document. A single key/value pair is a special case we can construct directly. But in general users should think of it as a list of key value pairs.

```
using namespace bsoncxx::builder;
list::document list_builder = {{"hello", "world"}, list::document{ {"nested", "doc"} };
```

Otherwise, users might get the wrong idea and think that they can add to this initializer list directly to add fields.

Reply:
+1